pkg/logger/README.md

```diff
@@ -1,40 +1,174 @@
-# logger package
+# Logger Package
 
-The logger package is intended to make unified log management in the whole app
+A structured logging package built on top of [uber-go/zap](https://github.com/uber-go/zap), providing both global and context-aware logging capabilities with sensible defaults.
 
-The log may be used in 2 ways:
+## Features
 
-- Global: easy use of the global logger without the need to init or configure. 
+- Built on top of the high-performance zap logger
+- Supports both structured and printf-style logging
+- Context-aware logging
+- Global and instance-based logging
+- Configurable log levels and encoding formats
+- Request ID tracking support
+- Easy integration with existing applications
+
+## Installation
+
+```go
+go get -u go.uber.org/zap
```

Comment:
go get zap? I don't think this is correct!

Reply:
You are right.  Fixed